src/components/PostDetails.tsx

```diff
@@ -98,21 +62,21 @@ export const PostDetails: React.FC<Props> = ({ post }) => {
       </div>
 
       <div className="block">
-        {!loaded && <Loader />}
+        {!loading && <Loader />}
```

Comment:
The logic for displaying the `Loader` seems inverted. Currently, it shows the `Loader` when `!loading` is true, which means loading is false. Typically, a loader should be displayed when `loading` is true. Consider revising this condition to `{loading && <Loader />}`.